rules/concepts.md

```diff
@@ -194,8 +198,504 @@ Available types of conditions include:
 - An Item has a given state
 - It’s a certain time of day
 - It’s a certain day of week
-- It’s a special day, e.g. holiday, weekday, weekend, etc. as defined by Ephemeris <!-- TODO: Add reference to Ephemeris docs which currently reside in the Actions page --> 
+- It’s a special day, e.g. holiday, weekday, weekend, etc. as defined by Ephemeris <!-- TODO: Add reference to Ephemeris docs which currently reside in the Actions page -->
```

Comment:
Before we merge merge and submit this to upstream we need to find and fix all the remaining todos.

Reply:
Yes, and there are many TODOs, I‘ve just added two more.